frontend/src/app/core/helpers/default-schedule-helper.ts

```diff
@@ -2,7 +2,8 @@ import { getScheduleItems } from '@core/helpers/schedule-list-helper';
 import { ISchedule } from '@core/models/schedule/ISchedule';
 
 export const getDefaultSchedule = (externallyDefined: boolean): ISchedule => ({
-    timeZone: 0,
+    timeZoneName: 'Europe/Kiev (+03:00)',
```

Comment:
it would be better to show current user time zone

Reply:
We can show current user time zone by [getUserZone]="true" on html (as in external-booking-choose-time-page.component). But if we use it on schedule component we hide a time zone data from db for existing schedules